src/containers/Tenant/ObjectSummary/transformPath.ts

```diff
@@ -14,3 +19,19 @@ export function transformPath(path: string, dbName: string): string {
 
     return result;
 }
+
+export function isDomain(path: string, type?: EPathType) {
```

Comment:
import { isDomain } from './path-utils';
import { EPathType } from './types';

describe('isDomain', () => {
    it('should return true for valid domain paths', () => {
        expect(isDomain('domain/', EPathType.EPathTypeDir)).toBe(true);
        expect(isDomain('another-domain/', EPathType.EPathTypeDir)).toBe(true);
    });

    it('should return false for non-directory paths', () => {
        expect(isDomain('domain/', EPathType.EPathTypeFile)).toBe(false);
        expect(isDomain('domain/', undefined)).toBe(false);
    });

    it('should return false for paths without trailing slash', () => {
        expect(isDomain('domain', EPathType.EPathTypeDir)).toBe(false);
    });

    it('should return false for paths with multiple slashes', () => {
        expect(isDomain('domain/subdomain/', EPathType.EPathTypeDir)).toBe(false);
        expect(isDomain('/domain/', EPathType.EPathTypeDir)).toBe(false);
    });

    it('should return false for empty paths', () => {
        expect(isDomain('', EPathType.EPathTypeDir)).toBe(false);
    });

    it('should return false for root path', () => {
        expect(isDomain('/', EPathType.EPathTypeDir)).toBe(false);
    });
});